foodnet/membership/models.py

```diff
@@ -53,6 +53,18 @@ def is_complete(self):
             return True
         return False
 
+    def has_admin_permission(self, department):
+        return self.departmentadministrator_set.filter(department=department).exists()
+
+    def can_be_edited_by(self, user_profile):
+        # can be edited by user_profile if the user_profile is an admin of
+        # the self.member.department department
+        departments = self.member.department_set.all()
+        for department in departments.iterator():
```

Comment:
I'll remove it. I think that I found out about iterator() just by exploring the model in a ipython session.